### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by stramel** [angular-ui/AngularJS-Atom#22]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/usr/local/share/atom/resources/app/src/package.js:565:9)
<unknown> (/usr/local/share/atom/resources/app/src/package.js:187:43)
```

**Atom:** 0.189.0-edd46e4

**Comment by outsideris** [comment]
Thank you for reporting. I will check it.

**Comment by outsideris** [comment]
It already fixed in @716739a
